providers/implementations/storemgmt/file_store.c

```diff
@@ -450,13 +450,11 @@ static int file_setup_decoders(struct file_ctx_st *ctx)
              * attached to any provider, since it's only used locally.
              */
             to_obj = ossl_decoder_from_algorithm(0, to_algo, NULL);
-            if (to_obj == NULL)
+            if (to_obj != NULL)
+                to_obj_inst = ossl_decoder_instance_new(to_obj, ctx->provctx);
+            OSSL_DECODER_free(to_obj);
+            if (to_obj_inst == NULL)
```

Comment:
And to_obj_inst has to be initialized to NULL at the declaration.

Reply:
Good point!